webpack.config.js

```diff
@@ -1,12 +1,13 @@
 const config = {
   devtool: "cheap-source-map",
-  entry: ['./src/index.js'],
+  entry: ['babel-polyfill', './src/index.js'],
```

Comment:
why did you add the polyfill here?

Reply:
Couldn't get the regenerator-runtime stuff to work without it... will try again a little today to see if I can remove it

Reply:
I tried and removed the polyfill, but used it in the code where I use this lib:

```
import 'babel-polyfill';
const { createSignMessage } = require('@lunie/cosmos-api');
```

Reply:
Ah yeah that makes sense. Will update this PR later today